gitpod-io/gitpod
# [dashboard] Add loading indicator to Prebuilds page

## Description


Adds a loading indicator to the Prebuilds page.

Also drive-by disables an annoying ESLint rule, and moves a component into the correct directory.

## Related Issue(s)

Fixes https://github.com/gitpod-io/gitpod/issues/5351

## How to test


1. Open the Prebuilds page of any project, briefly notice the new loading indicator

## Release Notes

```release-note
[dashboard] Add loading indicator to Prebuilds page
```

## Documentation


/uncc
